### ISSUE TITLE
colons : in enum values cause declaration errors

### ISSUE THREAD

**Issue by dimitri-yatsenko** [datajoint/datajoint-matlab#34]
For example, in Jake's `patch.Session`, he declared

```
craniotomy_location=null    : enum('V1: 2.7-3 lateral, just anterior of lambda','S1: 3.2-3.5 lateral, 1.2-1.5 posterior (bregma)') # 
```

This throws an error because `:` is used to separate attribute name from the datatype.

**Comment by eywalker** [comment]
What's the status on this? This sounds like a rather annoying bug that ought to be fixed soon.

**Comment by dimitri-yatsenko** [comment]
Fixed in #72.
